nucleus/pom.xml

```diff
@@ -289,10 +289,11 @@
 
 
         <!-- Other -->
-
-        <hk2.version>2.5.0-b61.payara-p1</hk2.version>
+        <hk2.version>2.5.0.payara-p1</hk2.version>
         <hk2.plugin.version>2.5.0-b61.payara-p2</hk2.plugin.version>
```

Comment:
Does this have a new version as well?

Reply:
The plugin version is separate from the hk2 version and are releases independently. As long as the plugin has not been changed, this does not need to be updated.

Reply:
The plugin does however have a new version, so I just wanted to check if it was appropriate to update it

Reply:
updated the plugins version as well.